dockstore-webservice/src/main/resources/migrations.1.6.0.xml

```diff
@@ -343,4 +343,9 @@
             UPDATE sourcefile SET absolutepath = CONCAT('/', absolutepath) WHERE absolutepath NOT LIKE '/%'
         </sql>
     </changeSet>
+    <changeSet id="checkAbsolutePath" author="gluu">
+        <sql dbms="postgresql">
+            ALTER TABLE sourcefile ADD CONSTRAINT absolutepath_check CHECK (absolutepath LIKE '/%' OR absolutepath='')
```

Comment:
Has this been tested with the most recent censored dump?

Reply:
just tested again with:

```
psql webservice_test < ~/Downloads/ds-webservice*sql -U dockstore
java -jar ~/dockstore/dockstore-webservice/target/dockstore-webservice*SNAPSHOT.jar db migrate -i 1.5.0,1.6.0 ~/.dockstore/dockstore.yml
```

and ran webservice with validate, it seems to not complain.